kedro-datasets/kedro_datasets/ibis/table_dataset.py

```diff
@@ -119,19 +126,29 @@ def __init__(  # noqa: PLR0913
 
         # Set load and save arguments, overwriting defaults if provided.
         self._load_args = deepcopy(self.DEFAULT_LOAD_ARGS)
-        if load_args is not None:
+        if load_args:
             self._load_args.update(load_args)
         if database is not None:
             self._load_args["database"] = database
 
         self._save_args = deepcopy(self.DEFAULT_SAVE_ARGS)
-        if save_args is not None:
+        if save_args:
```

Comment:
I think this is technically slightly more efficient. IIRC I tried to be consistent with the other dataset implementations; maybe it makes sense to undo this for now, and potentially (separately) just update all the datasets to do this?